src/main/resources/assets/api-docs.yaml

```diff
@@ -2325,6 +2325,35 @@ paths:
           description: The Dataset was deleted.
         500:
           description: Server error.
+  /api/dataset/{datasetId}:
+    put:
+      summary: Updates the Dataset from JSON
+      description: Updates the Dataset from JSON
```

Comment:
Suggestion from previous comment, we should fully describe the required properties here in an expanded description.